snet-local-env/docker_run.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+docker run -p 5002:5002 -p 8545:8545 -p 7000:7000 -ti singularitynet/snet-local-env:latest
```

Comment:
The `latest` here is not matching the `TAG=2.0.2` in `docker_build.sh`

Reply:
I'd like also to suggest naming this container using `--name`.

Reply:
Fixed TAG issue

Reply:
> I'd like also to suggest naming this container using --name snet_local_dev (or something else).

I used `--name` at the beginning but in such case when one calls `start.sh` twice `docker` claims that such container is already exist. It is possible to write more sophisticated script which will check whether container exists and started and do different things but I am not sure it is needed. Another workaround is to use `--name` and `--rm` but then there is a risk of container lost after user leaves the shell. I am not sure what to do with it.

Reply:
> calls start.sh twice docker claims that such container is already exist.

But this is the expected behavior, right? I mean, why an user will need a second container?

A named container is cool because user can start/stop/remove it with ease.

Reply:
But I think we can go without `--name` too.

Reply:
After I started using it I found that every time I rebuild image I need to delete previous container first to start it again and I removed `--name` :-) Ok, may be when one not rebuilds it each time it makes more sense. Ok I think I can return `--name` and add possibility to add other docker options to the command tail.